src/components/Sum/Sum.jsx

```diff
@@ -1 +1,9 @@
 // export const Sum = () => ();
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
```

Comment:
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```
You can place it in one line, as there are < 3 props and the line is not too long